src/Handler/CurlFactory.php

```diff
@@ -375,6 +375,14 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf)
             $conf[CURLOPT_TIMEOUT_MS] = $options['timeout'] * 1000;
         }
 
+        if (isset($options['force_ipv4'])) {
+            if (defined('CURLOPT_IPRESOLVE') && defined('CURL_IPRESOLVE_V4')) {
```

Comment:
So if this feature comes with every version of curl and php, why do you need to check it's existence? Just two lines bellow another curl constant is used and it doesn't check curl existence, not to mention that it is CURL factory so it might probably be clear by this time if curl is present or not.